### PR TITLE
Support upload slow trace

### PR DESCRIPTION
**Pull request by mrproliu**
Following https://github.com/apache/skywalking/issues/9802, Support uploads the slow request with tracing context.

### CODE REVIEW

**Comment by wu-sheng** [comment]
One question, what are we left for the HTTP content sampling in profiling?

We have
- [x] HTTP/1 topology
- [x] HTTP/1 metrics for topology lines and edges
- [x] Slow trace sampling in this PR.

I think we don't have attachedEvent for span yet. What do I miss?

**Comment by mrproliu** [comment]
Nothing is missing. 
For the HTTP1 analysis, we now have HTTP1 metrics and top sampled slow traces. but we cannot query the full HTTP content with tracing context(the attached event for the span is not implemented).
After the attached events for the span are implemented, I will extend the L2-L4 event for the tracing span.

File: pkg/profiling/task/network/analyze/layer7/protocols/tracing.go [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+		return analyzeSkyWalking8TracingContext(sw8Header)
+	}
+
+	// zipkin
+	if zipkinTraceID := fetcher("x-b3-traceid"); zipkinTraceID != "" {
+		if spanID := fetcher("x-b3-spanid"); spanID != "" {
+			return analyzeZipkinTracingContext(zipkinTraceID, spanID), nil
+		}
+	}
+	return nil, nil
```
Comment (wu-sheng) [inline pkg/profiling/task/network/analyze/layer7/protocols/tracing.go:66]: I remember there is another `b3` header format of Zipkin. Could you check?

Comment (wu-sheng) [reply pkg/profiling/task/network/analyze/layer7/protocols/tracing.go:66]: Both two should be legal.

Comment (wu-sheng) [reply pkg/profiling/task/network/analyze/layer7/protocols/tracing.go:66]: Check here, https://github.com/openzipkin/b3-propagation#single-header

**Comment by wu-sheng** [comment]
> After the attached events for the span are implemented, I will extend the L2-L4 event for the tracing span.

OK, once this is done, ebpf relative features in 9.3.0 milestone should be ready, right? Including rover side.

**Comment by mrproliu** [comment]
> > After the attached events for the span are implemented, I will extend the L2-L4 event for the tracing span.
> 
> OK, once this is done, ebpf relative features in 9.3.0 milestone should be ready, right? Including rover side.

Yes. The rover side could analyze the HTTP1 metrics, slow traces, and extend the tracing context. The OAP side could set up a new dashboard for HTTP1 in network profiling and query the attached event on the tracing page.

**Comment by wu-sheng** [comment]
Please update the relative network profiling documents.

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**